src/server/src/main/java/io/cassandrareaper/storage/MemoryStorageFacade.java

```diff
@@ -127,4 +178,126 @@ public IClusterDao getClusterDao() {
     return this.memClusterDao;
   }
 
-}
\ No newline at end of file
+  private void persist(Object... objects) {
+    synchronized (memoryStorageRoot) {
+      try {
+        this.embeddedStorage.storeAll(objects);
+        this.embeddedStorage.storeRoot();
+      } catch (Throwable ex) {
+        ex.printStackTrace();
```

Comment:
**Todo:** This should be replaced by a call to the logger
```suggestion
        LOG.error("Failed persisting Reaper state on disk", ex);
```

Reply:
Same comment here, the stack trace was just making it easier to track down what was failing to persist. Now that I've fixed it, we probably don't need the try-catch at all.

Reply:
Catching RuntimeExceptions is probably a good thing to do so we can have an error message that's easy to read.
wdyt?

Reply:
Ok, I can leave it in. Will change it from Throwable to RuntimeException.